src/bridging/providers/across/const/tokens.ts

```diff
@@ -0,0 +1,68 @@
+import { TargetChainId, SupportedChainId, AdditionalTargetChainId } from '../../../../chains/types'
+
+/**
+ * Chain config for Across. Includes all the supported tokens for the chain.
+ */
+export interface AcrossChainConfig {
+  chainId: TargetChainId
+  tokens: { [name: string]: string | undefined }
+}
+
+const ACROSS_CHAIN_CONFIGS: AcrossChainConfig[] = [
```

Comment:
TODO: fetch dynamically?

Reply:
Yes, we should, but this is out of the scope. This implementation provides bridging for the known networks and tokens in the config. 

This will get us to test the SDK, provider, etc. 

To iron the providers we will want to work with the bridge providers to make sure we make the most of their protocols.